docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -29,6 +29,7 @@ A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute)
 - **ORDER BY**
 
 The **ORDER BY** operator is supported only when Elasticsearch is used as the Visibility store.
+This means that the **ORDER BY** operator is currently not supported in Temporal Cloud.
```

Comment:
Not needed for this PR, but I wonder if we want to just remove (or bury into a spot at the bottom of this page) anything about order by if it's not supported in cloud.

Reply:
@cretz Ya, I have been thinking about how to more broadly indicate which features / things are supported across OSS vs Cloud.

If you have any suggestions - lets talk!

Reply:
No suggestions in particular beyond burying/de-emphasizing the non-cloud features rather than burying/de-emphasizing a statement that a feature is non-cloud. But that's more general, PR LGTM but may still want a server team approver.